spec/spec_helper.rb

```diff
@@ -1,4 +1,20 @@
 require 'pathname'
+
+require 'simplecov'
+
+SimpleCov.start do
```

Comment:
Is there an advantage to putting the config and start here, rather than in '.simplecov.'? It gets loades when simplecov and is required, so no churn in spec helpers

I've also found it best to leave off when not running whole suite as it can slow down the prog exit